source/features/view-markdown-source.tsx

```diff
@@ -98,7 +98,7 @@ async function init(): Promise<void> {
 	const fileButtons =
 		select('.repository-content .Box-header.flex-md-items-center .d-flex') ??
 		// Pre "Repository refresh" layout
-		select('.repository-content .Box-header .d-flex')!;
+		select('.repository-content readme-toc .Box-header .d-flex')!;
```

Comment:
> Pre "Repository refresh" layout

I don’t think this is the right part of the code

Reply:
That's what I thought too but the first selector doesn't match anything in the DOM, while the second does.

Reply:
Wow, how can we have the same identical issue a year later? If the pre/post refresh code is creating issues, lets just drop it in this case.